introspection/org.freedesktop.NetworkManager.Device.OvsBridge.xml

```diff
@@ -7,6 +7,16 @@
   -->
   <interface name="org.freedesktop.NetworkManager.Device.OvsBridge">
 
+    <!--
+        Slaves:
+
+        Array of object paths representing ports which are currently slaved to
+        this bridge.
```

Comment:
slaved -> enslaved?

Reply:
Yes. I've corrected it in some places while cargo-culting, lazy to fix the originals, and left some oversights. Added a commit that fixes the rest now.